directdemod/merger.py

```diff
@@ -92,19 +92,23 @@ def merge(self, jsons, whole=False, resolution=constants.RESOLUTION):
         self.top_ex   = -90
 
         for obj in jsons:
-            image  = mimg.imread(obj["image_name"])
+            image, isGray = self.imread(obj["image_name"])
             center = obj["center"]
             degree = obj["direction"]
 
             if self.is_cartopy:
-                img = self.set_transparent(ndimage.rotate(image, degree, cval=255))
+                img = self.set_transparent(ndimage.rotate(image, degree, cval=255), isGray)
                 dx = img.shape[0]*4000/2*0.81 # in meters
```

Comment:
This arbitrary 0.81 still remains even after georeferencing? or this is something else?

Reply:
I have written in log file, that merger isn't yet made to support plotting of tiff files. Because I thought about using something more powerful. So in this PR merger hasn't been changed, I only added support of gray images.